### DISCUSSION TITLE
Q: 客户端注册时提示fetch failed

### DISCUSSION THREAD

**Discussion by linxuesong** [walinejs/waline#2017] [Q&A] [answered]
### Describe the bug | 问题描述

**访问入口:**
https://snowday.cn/posts/%E6%96%87%E7%AB%A0%E6%A0%B7%E5%BC%8F%E6%B5%8B%E8%AF%95/

**问题:**
客户端注册提示fetch failed
服务端日志
Deprecated] /article API will be deprecated in the next major version, please don't use it anymore. If you are using `@waline/client` please upgrade to `@waline/client@3`. For other scenarios, you can use `/api/article` to replace it.
这个是需要升级服务端的包吗
**场景：** 独立部署到服务器，使用node node_modules/@waline/vercel/vanilla.js启动
主要配置
```
export SITE_URL=https://waline.snowday.cn
#export SECURE_DOMAINS=https://www.snowday.cn;https://waline.snowday.cn
```

### Website URL | 问题网站

https://snowday.cn

### Where your waline deploy? | 服务部署在哪里？

Other

### Where your comment data store? | 数据存储在哪里？

MySQL

**Comment by lizheming** [comment, marked as answer]
nginx 需要注意下这两个配置得有

```
proxy_set_header X-Forwarded-Proto $scheme;
proxy_set_header REMOTE-HOST $remote_addr;
```
